airbyte-integrations/connectors/destination-s3/finalize_build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -e
+
+ARCH=$(uname -m)
+
+if [ "$ARCH" == "x86_64" ] || [ "$ARCH" = "amd64" ]; then
+  echo "$ARCH"
+  yum install lzop lzo lzo-dev -y
+
+# alanechere: I'm not sure we need this custom install of lzo anymore. Using the yum install above works in the build context.
```

Comment:
@edgao I was not able to run this script successfully. The `sdk install java 8.0.342-librca` command exits with a 1 status code. Considering the current code freeze on this connector I think it's fine to merge this PR with a "broken" script. I'll let your team figure the changes required to support LZO on all architectures. I'm even not sure the custom install for ARM64 is required. I tried a simple  `yum install lzop lzo lzo-dev -y` on the ARM64 variant and it worked.

You can reproduce my failure on this branch by running:
`airbyte-ci connectors --name=destination-s3 build`

I'll also let you decide if you prefer to use a bash script or the `finalize_build.py` module with a `finalize_build` (please check the `destination-iceberg/finalize_build.py` for an example). The 🐍 way is better for improved caching and access to the Dagger API. But I also understand that running this script as it is is convenient.

Reply:
amazing. yeah I'll just submit an issue to figure this out, not a blocker for this pr.